docs/dev/Kibana-Reporting-Design-Proposal.md

```diff
@@ -50,7 +50,34 @@ Kibana Reports for Open Distro allows ‘Report Owner’ (engineers, including b
         * As a user, I should be able to archive reports. 
             * Delete option should not be there as it might lead to complications such as one wherein the user who receives the reports deletes them and says that they never received the reports
         * As a user, I should be able to see a list of reports archived for the past 100 days. NOTE: Reports will be stored as hyperlinks/urls and NOT as actual reports. If the data is available at the time when the url is clicked, report will be created and can be viewed in a browser and can be downloaded.
-
+    * For Saved Search CSV Report
+        * As a user, I should be able to find my saved search through an autocomplete input, in order to find my saved search quickly.
+        * As a user, I should be able to choose time range for my csv data generation. This time range should be by default the same value of previous chosen time range from other tabs
```

Comment:
Can you specify what you mean by "same value of previous chosen time range?" What if I am creating a Saved Search CSV report for the first time?

Reply:
Time range picker exists in different tabs in Kibana such as Discover, Visualizations, Dashboard, Reporting. It should keeps the time range value between all tabs. For example, user set time range picker at "last 7 days" in discover tab, then we should have automatically the same value "last 7 days" when the user arrives in Reporting tab.
 
```suggestion
        * As a user, I should be able to choose time range for my csv data generation. Time range picker value should be shared between tabs in Kibana.
```

Reply:
`should be shared between tabs` -> `we can inherit the time range from existing tabs in Kibana` ?